tc-vars.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -ex
+
+HOME_CLEAN=$(/usr/bin/realpath "${HOME}")
+
+PATH=${HOME_CLEAN}/bin/:$PATH
+export PATH
```

Comment:
I realise this isn't newly introduced by this patch, but is there any reason to not just do this on one line? (same comment for each subsequent case of this)

If it was about saving space, I'd expect all the variables first, then a single export line at the end, e.g.:
```
VAR1=foo
VAR2=bar
VAR3=baz

export VAR1 VAR2 VAR3
```

If this is just a matter of taste, I have no objection as it's quite unlikely I'll be doing much maintenance in this area :)